MyBankAccountProject/MyBankAccountProject/BankAccountProject.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MyBankAccountProject
+{
+    public enum AccountState { Opened, Closed }
+    class BankAccountProject
+    {
+        private decimal balance = 0;
+        private Person client;
+        public List<CompleteTransaction> ListOfTransactions { get; set; } = new List<CompleteTransaction>();
+        public BankAccountProject(Person client)
+        {
+            this.client = client;
+        }
+        public void InfoAboutPerson()
+        {
+            Console.WriteLine("Information for creating a bank account:");
+            Console.WriteLine($"Firstname:{client.FirstName}");
+            Console.WriteLine($"Lastname:{client.LastName}");
+            Console.WriteLine($"Cnp:{client.Cnp}");
+            Console.WriteLine($"Address:{client.Address}");
+            foreach (var completeTransaction in ListOfTransactions)
+            {
+                Console.WriteLine($"Transaction of type: {completeTransaction.Transaction.TypeTransaction} \nValue: {completeTransaction.Transaction.AmountTransaction},\nDate:{completeTransaction.Transaction.DateTransaction},\nDetails:{completeTransaction.Transaction.DetailTransaction}");
+                Console.WriteLine($"Comision: {completeTransaction.Comision.Value}");
+            }
+        }
+        public void OpenAccount()
+        {
+            Console.WriteLine($"The account is {AccountState.Opened}!");
```

Comment:
si unde mentii starea contului? Daca apelezi metoda, se scrie un mesaj in consola, dar....starea obiectului, ca e opened...nu o tii nicaieri. Daca vrei sa inchizi contul...de unde stii ca e deschis sau nu?